<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Secret</class>
 <widget class="QMainWindow" name="Secret">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonPlay">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#buttonPlay:hover{background-color:rgba(0,0,0,0.5);}
QPushButton#buttonPlay{background-color:rgba(0,0,0,0);border-radius:15px;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="commandeButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#commandeButton:hover{background-color:rgba(0,0,0,0.5);}
QPushButton#commandeButton{background-color:rgba(0,0,0,0);border-radius:15px;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="creditButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#creditButton:hover{background-color:rgba(0,0,0,0.5);}
QPushButton#creditButton{background-color:rgba(0,0,0,0);border-radius:15px;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="top10Button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#top10Button:hover{background-color:rgba(0,0,0,0.5);}
QPushButton#top10Button{background-color:rgba(0,0,0,0);border-radius:15px;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="leaveButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#leaveButton:hover{background-color:rgba(0,0,0,0.5);}
QPushButton#leaveButton{background-color:rgba(0,0,0,0);border-radius:15px;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listLevel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
